movies-manager/src/styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+$grey: #474a51;
```

Comment:
If you will store in this file only colors variables than name it ass colors-vars.sccs

Reply:
done